### ISSUE TITLE
Renewer STDERR logging

### ISSUE THREAD

**Issue by bmw** [letsencrypt/letsencrypt#988]
I haven't had the time yet to investigate this in more detail, but I just experienced the following behavior. I:

* From a clean LE install, ran `letsencrypt -t`. Nginx was not installed so Apache was automatically chosen. I received and installed my cert.
* I added these lines to `/etc/letsencrypt/renewer.conf`:
```
renew_before_expiry = 10 years
deploy_before_expiry = 10 years
```
* I ran `letsencrypt-renewer`.

After doing this, messages at every log level were output to STDERR. This is not expected or desired behavior.

**Comment by SwartzCr** [comment]
I'm going to take a look at this

**Comment by SwartzCr** [comment]
Now that I have gotten the code to read our config file I have reproduced this error

**Comment by SwartzCr** [comment]
fix here #1941

**Comment by bmw** [comment]
Fixed in #1941.
